core/trino-main/src/main/java/io/trino/sql/planner/optimizations/HashGenerationOptimizer.java

```diff
@@ -872,14 +869,10 @@ public HashComputationSet withHashComputation(Optional<HashComputation> hashComp
             return Optional.empty();
```

Comment:
You stop using `HashGenerationOptimizer.INITIAL_HASH_VALUE` but it is still used in `IntepretedHashGenerator`. If somebody changes this constant, the hashes will become out-of-sync.